### PR TITLE
Milestone #2

### PR DESCRIPTION
**Pull request by borispovod**

### CODE REVIEW

File: deliveries/pontem-network_milestone-2.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Milestone Delivery :mailbox:
+
+**The [invoice form :pencil:](https://forms.gle/8Wx7nxtq8fKrsuEz8) has been filled out correctly for this milestone and the delivery is according to the official [milestone delivery guidelines](https://github.com/w3f/General-Grants-Program/blob/master/grants/milestone-deliverables-guidelines.md).**  
+
+* **PR Link:** [Pontem Network](https://github.com/w3f/Open-Grants-Program/pull/138).
```
Comment (olegvg) [inline deliveries/pontem-network_milestone-2.md:5]: Seems wrong link. It pointed to Milestone #1 PR

File: deliveries/pontem-network_milestone-2.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+* **Milestone Number:** [Milestone #2](https://github.com/w3f/Open-Grants-Program/blob/master/applications/pontem.md#milestone-2--alpha-version-of-move-pallet).
+
+See how to try result of work on milestone #2: Move VM, Move Pallet, and tools, in Move Pallet [documentation](https://github.com/dfinance/sp-move/blob/master/README.md).
+
+| Number | Deliverable | Link | Notes |
+| ------------- | ------------- | ------------- |------------- |
+| 0-1. | Adopt storage - WriteSets processing | [Move Pallet](https://github.com/pontem-network/sp-move/commit/45a2cfb4d33db5f4a5792b43de313b313d3ec3ca)  | Move Pallet saving result of transaction execution in pallet storage. We still use LCS codec instead of Polkadot one, as migration is too expensive. |
+| 2. | Events processing | [Move Pallet Commit #1](https://github.com/pontem-network/sp-move/commit/45a2cfb4d33db5f4a5792b43de313b313d3ec3ca) [Move Pallet Commit #2](https://github.com/pontem-network/sp-move/commit/d74a74801bf5f6f24096ad86df929708f3c39d14) | Supporting events emits from smart contracts and write result events in storage. Also, events format compatible with original diem ones.  |
```
Comment (olegvg) [inline deliveries/pontem-network_milestone-2.md]: Supporting events ... -> Events generated by smart contracts are written to storage now. Their format is compatible with Diem.

File: deliveries/pontem-network_milestone-2.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+| 5. | Standard Library	| [Move Pallet](https://github.com/pontem-network/sp-move/commit/d74a74801bf5f6f24096ad86df929708f3c39d14) [Standard Library](https://github.com/pontem-network/move-stdlib) | Publish Move standard library via transaction from root account. Also, the standard library version implementation for pallet, supporting functional like: block height, time, balances, signatures etc. |
```
Comment (olegvg) [inline deliveries/pontem-network_milestone-2.md]: Also, the standard library version ... -> Also, the standard library variant for pallet provides an access to block height, time, balances, signatures etc.

File: deliveries/pontem-network_milestone-2.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+| 6. | Execute Transaction | [Move Pallet](https://github.com/pontem-network/sp-move/blob/master/pallets/sp-mvm/src/lib.rs#L70) [Move Pallet Commit #1](https://github.com/pontem-network/sp-move/commit/f0781c98543b1d5afc3a35f5041cb116438516c4#diff-4e5930d2e2ba9c4d86fb707cc5d1eb7aef9839019ea8e6f3125310aa710557cbR189) | Support for transaction contains Move scripts to execute.  |
```
Comment (olegvg) [inline deliveries/pontem-network_milestone-2.md]: Transactions containing Move script are recognized and executed now.

File: deliveries/pontem-network_milestone-2.md (hunk 1 above)
Comment (borispovod) [reply deliveries/pontem-network_milestone-2.md:5]: It's link to initial grant pull request, next link points to milestone #2, so all fine

**Pull request merged**